js/main.js

```diff
@@ -0,0 +1,27 @@
+$(document).ready(function () {
+  var myInit = { method: 'GET',
+               cache: 'default' };
+
+  fetch('https://cors-anywhere.herokuapp.com/https://api.meetup.com/slcdevs/events?&amp;sign=true&amp;photo-host=public&amp;page=20', myInit)
```

Comment:
Can we URL encode the second `https://` - I know this shouldn't matter for most browsers, but want to make sure we optimize for compatibility